src/pages/Integrations/Create/CreatePage.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import { Messages } from '../../../properties/Messages';
+import {
+    Button,
+    Form,
+    FormGroup,
+    FormSelect,
+    FormSelectOption,
+    Modal,
+    TextInput
+} from '@patternfly/react-core';
+import { useEffect, useState } from 'react';
+import { Integration, IntegrationType } from '../../../types/Integration';
+
+const EMPTY = '';
+const choose = Messages.common.choose;
+const webhookType = Messages.pages.integrations.types.hooks;
+const initialValue='(enter hooks url)';
+
+interface CreatePageProps {
+    isModalOpen: boolean;
+    model: Integration[];
+    updateModal: (isOpen: boolean) => void;
+    updateModel: (integrations: Array<Integration>) => void;
+    currentRow?: string;
+    updateCurrentRow?: (currentRowOpen: string) => void;
+}
+
+const CreatePage: React.FunctionComponent<CreatePageProps> = props => {
+
+    const [pageModal, updatePageModal] = useState(props.isModalOpen);
+    const [newUrl, updateUrl] = useState(initialValue);
+    const [intCount,updateIntCount] = useState(0);
+    const [currentRow,updateCurrentRow] = useState(props.currentRow);
+    const [pageTitle,updatePageTitle] = useState(Messages.components.integrations.toolbar.actions.addIntegration);
+    useEffect(() => {
+        if(props.currentRow){
+            updatePageTitle(Messages.components.integrations.toolbar.actions.editIntegration);
+            let integration = props.model.find(element => element.id == props.currentRow);
+            //preload url for editing if available.
+            if(integration){
+                updateUrl(integration.url);
```

Comment:
`formik` would fill this (and any other data that we have), it would be good to use it (in a separate PR), as this page will be more complex when more types appears.

Reply:
formik can do this too? Hmm.. I'm still reading up on it now. I'll look to address that in a another PR then.